keys/cryptostore/holder.go

```diff
@@ -25,11 +25,13 @@ func New(coder Encoder, store keys.Storage, codec keys.Codec) Manager {
 }
 
 // exists just to make sure we fulfill the Signer interface
+// nolint [ megacheck, deadcode ]
 func (s Manager) assertSigner() keys.Signer {
```

Comment:
instead of these functions, we can make empty var declarations to assert interface compliance, eg. 

```
var _ keys.Signer = Manager{}
var _ keys.Manager = Manager{}
```

should do the trick

Reply:
didn't do the trick

Reply:
you can remove the assert functions once you add the var declarations :)

I got it